BunqSdk.Tests/Model/Generated/Endpoint/TabUsageSingleTest.cs

```diff
@@ -16,21 +16,21 @@ public class TabUsageSingleTest : BunqSdkTestBase
         /// <summary>
         /// Config values
         /// </summary>
-        private const string TAB_FIELD_DESCRIPTION = "Pay the tab for Java test please.";
-        private const string FIELD_STATUS_OPEN = "OPEN";
-        private const string AMOUNT_EUR = "10.00";
-        private const string FIELD_CURRENCY = "EUR";
-        private const string TAB_ITEM_FIELD_DESCRIPTION = "Super expensive java tea";
-        private const string FIELD_STATUS_WAITING = "WAITING_FOR_PAYMENT";
+        private const string TabUsageSingleDescription = "Pay the tab for Java test please.";
+        private const string TabUsageSingleStatusOpen = "OPEN";
+        private const string TabUsageSingleAmountEur = "42.00";
```

Comment:
:open_mouth: `42.00` is way to much :P it will drain the test account pretty fast haha! Please lower this amount.